packages/docs/components/CollapsibleParams.tsx

```diff
@@ -0,0 +1,14 @@
+interface CollapsibleParamsProps {
+  children: React.ReactNode;
+}
+
+export default function CollapsibleParams({
```

Comment:
I think for PR's that include UI changes it would be helpful to see screenshots in the PR description.